src/patterns/components/tables/react/code/secondary-row-comparison.hbs

```diff
@@ -0,0 +1,13 @@
+  <SprkTable 
+    variant="secondaryRowComparison"
+    additionalTableClasses="sprk-b-Table--spacing-medium"
+    idString="table-5"
+    columns={[ {name: 'data1'}, {name: 'data2'} ]}
```

Comment:
I think the outer object may not be necessary here.

Reply:
The outer curly braces are so I can pass an array as a prop.

Reply:
haha wow, I'm sorry, you're right. I got really confused lol. Carry on...